.devcontainer/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
```

Comment:
@SteveL-MSFT, your last commit had 8 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

NullConditionalMemberAccess.?. operator tests.Can get member value of a non-null variable
```powershell
Expected like wildcard 'pwsh*' to match 'ConsoleHost mai', but it did not match.
at <ScriptBlock>, : line 5
5:             (Get-Process -Id $PID)?.Name | Should -BeLike "pwsh*"
```
NullConditionalMemberAccess.?. operator tests.Use ?. on a dynamic property name
```powershell
Expected like wildcard 'pwsh*' to match 'ConsoleHost mai', but it did not match.
at <ScriptBlock>, : line 2
2:             (Get-Process -Id $PID)?.$propName | Should -BeLike 'pwsh*'
```
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @(@{InputObject=System.Buffers.dll; SideIndicator=<=}, @{InputObject=System.Diagnostics.Debug.dll; SideIndicator=<=}, @{InputObject=System.Resources.ResourceManager.dll; SideIndicator=<=}, @{InputObject=System.Runtime.Extensions.dll; SideIndicator=<=}, @{InputObject=System.Threading.Tasks.dll; SideIndicator=<=}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 125
125:         $diffs | Should -BeExactly $null
```
NullConditionalMemberAccess.?. operator tests.Can get member value of a non-null variable
```powershell
Expected like wildcard 'pwsh*' to match 'ConsoleHost mai', but it did not match.
at <ScriptBlock>, : line 5
5:             (Get-Process -Id $PID)?.Name | Should -BeLike "pwsh*"
```
NullConditionalMemberAccess.?. operator tests.Use ?. on a dynamic property name
```powershell
Expected like wildcard 'pwsh*' to match 'ConsoleHost mai', but it did not match.
at <ScriptBlock>, : line 2
2:             (Get-Process -Id $PID)?.$propName | Should -BeLike 'pwsh*'
```